cmake/GetGitVersion.cmake

```diff
@@ -10,7 +10,8 @@
 # include(GetGitVersion)
 # get_git_version(GIT_VERSION)
 #
-# Requires CMake 2.6+
+# Requires CMake 2.8.11+
```

Comment:
I just noticed this version bump, but it was released in 2013. Commenting for posterity that it was noticed and approved.